### PR TITLE
feat: add approverInfo

### PR DESCRIPTION
**Pull request by nogahaskal**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ services:
     env_file:
       - ../kdrive.env
   user-service:
-    image: "drivehub.azurecr.io/meateam/user-service:v2.0.0"
+    image: "drivehub.azurecr.io/meateam/user-service:approverInfo"
```
Comment (Shahar-Y) [inline docker-compose.yml:81]: Use a version

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
1. Was it tested? and how? 
2. Does it affect other services?
3. Add a changelog.
4. Link to the deployment it was tested on.
Add all of this information in the PR description

File: docker-compose.yml (hunk 1 above)
Comment (nogahaskal) [reply docker-compose.yml]: done.

**Review by Shahar-Y — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+
+## Added
+
+- info about the approvers in the ui.
```
Comment (Shahar-Y) [inline CHANGELOG.md:13]: Add a link to the PR, describe if major/minor/hotfix

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+## [v2.0.2] - 2020-10-28
+
+### Removed
+
+- default timoout for the axios request because some big files uploads that took more then 10 sec faild.
+
+## [v2.1.0]
```
Comment (Shahar-Y) [inline CHANGELOG.md:21]: Those versions do not exist on github as versions. not related to the PR bot needs fixing

File: docker-compose.yml [hunk 4]
```diff
@@ -8,7 +8,7 @@ services:
     depends_on:
       - api-gateway
   api-gateway:
-    image: "drivehub.azurecr.io/meateam/api-gateway:v2.1.0"
+    image: "drivehub.azurecr.io/meateam/api-gateway:approverInfo"
```
Comment (Shahar-Y) [inline docker-compose.yml:11]: this tag should not exist in develop.

File: docker-compose.yml (hunk 1 above)
Comment (Shahar-Y) [reply docker-compose.yml:81]: mm.. not done?

File: src/components/popups/external-transfer-popup/steps/Approval.vue [hunk 5]
```diff
@@ -123,15 +159,29 @@ export default {
...
+        );
+
+        if (canApprove.canApproveToUser) {
+          this.selectedApprovals.push(approver);
+        } else if (canApprove.cantApproveReasons) {
+          this.blockedApprover = {
+            name: approver.fullName,
+            reasons: canApprove.cantApproveReasons,
+          };
+        }
```
Comment (Shahar-Y) [inline src/components/popups/external-transfer-popup/steps/Approval.vue:183]: What if (canApproveToUser && !cantApproveReasons)  ? shouldn't there be a default?

File: src/i18n/he.json [hunk 6]
```diff
@@ -150,14 +152,34 @@
...
+      "ApproverIsBlocked": "המשתמש חסום",
+      "ApproverIsNotInTheSameUnit": "שים לב הגורם שבחרת אינו גורם מאשר ביחידתך",
+      "ApproverCantApproveInHisUnit": "המאשר שבחרת אינו מוגדר כגורם מאשר",
+      "ApproverHasNoUnit": "המאשר שבחרת יכול לאשר העברת קבצים רק לעצמו (חסר יחידה)",
+      "UserHasNoUnit": "שים לב הינך חסר יחידה"
+    },
+    "cantApproveSupport": {
+      "Bam": "לפרטים נוספים אנא פנה ליחב\"ם",
+      "Bereshit": "ניתן לפתוח תקלה בtommy למול צוות בראשית",
+      "OpenInTomy": "לפתיחת תקלה"
```
Comment (Shahar-Y) [inline src/i18n/he.json:181]: Do these exist in the english version?

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Add a description and a link to the environment it was tested on

**Pull request merged**